public/javascripts/ng-apps.js

```diff
@@ -41,6 +41,23 @@ springInitiative.config(function($stateProvider, $urlRouterProvider, $locationPr
         'content': {
           templateUrl: 'views/content/student.html',
           controller: 'studentController'
+        },
+        'studentView@index.student': {
+          templateUrl: 'views/content/studentView.html'
+        }
+      }
+    })
+    .state('index.student.addEntry', {
```

Comment:
Do you have a documentation link showing that this is better? Just cuz I haven't seen that kind of naming in the tutorials i've looked at